src/main/webapp/WEB-INF/tags/proposaldevelopment/proposalDevelopmentAbstracts.tag

```diff
@@ -24,7 +24,18 @@
 <kul:tab tabTitle="Abstracts" defaultOpen="false" 
          tabItemCount="${fn:length(KualiForm.document.developmentProposalList[0].proposalAbstracts)}" 
          tabErrorKey="document.developmentProposalList[0].proposalAbstract*,newProposalAbstract*">
-         
```

Comment:
Since this is a JQuery .js function loaded at page load time we should avoid keeping the code for these embedded with the rest of the page. This should be at least at the top of the .tag file, if it can't be moved into a more generic .js like /kuali-coeus-5.2.1/src/main/webapp/scripts/ua_kuali_application.js..